index.html

```diff
@@ -1,527 +1,511 @@
-    <link
-      href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@100;200;300;400;500;600;700&family=Lato:wght@100;300;400;700;900&display=swap"
-      rel="stylesheet"
-    />
-  </head>
-  <body>
-    <!-- under construction banner -->
-    <div id="hbanner" class="under-construction">
-      <p class="wuc-paragraph">
-        Website under construction! We're currently building our website to
-        serve you better!
-      </p>
-      <button id="hbutton" class="under-construction-button">
-        <img
-          class
-          src="./assets/social-icons/material-symbols_close.svg"
-          alt="close under-construction banner"
-        />
-      </button>
-    </div>
-    <!-- header -->
-    <header>
-      <!-- wrapper -->
-      <div class="wrapper">
-        <!-- navigation -->
-        <nav class="header-nav">
-          <div class="logohead">
-            <a href="#home" aria-label="Return to homepage">
-              <img
-                src="./assets/images/logo-main-1.png"
-                alt="If These Lands Could Talk logo image"
-              />
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="title" content="If These Lands Could Talk - Preserving Tomorrow by Building Today" />
+  <meta name="description" content="If These Lands Could Talk (ITLCT) is global network of change-makers, supporters, and allies who support Native and Indigenous communities to thrive authentically and on their own terms with digital technology both now and in the future. We partner with communities and create aligned programming and events that amplify the voices, experiences, and excellence of Native and Indigenous populations throughout the world." />
+
+  <title>If These Lands Could Talk</title>
+  <link rel="stylesheet" href="./css/normalize.css" />
+  <link rel="stylesheet" href="./css/style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@100;200;300;400;500;600;700&family=Lato:wght@100;300;400;500;700;900&display=swap"
+    rel="stylesheet" />
+</head>
+
+<body>
+  <!-- under construction banner -->
+  <div id="hbanner" class="under-construction">
+    <p class="wuc-paragraph">
+      Website under construction! We're currently building our website to
+      serve you better!
+    </p>
+    <button id="hbutton" class="under-construction-button">
+      <img
+        class
+        src="./assets/social-icons/material-symbols_close.svg"
+        alt="close under-construction banner" />
+    </button>
+  </div>
+  <!-- header -->
+  <header>
+    <!-- wrapper -->
+    <div class="wrapper">
+      <!-- navigation -->
+      <nav class="header-nav">
+        <div class="logohead">
+          <a href="/" aria-label="Return to homepage">
```

Comment:
Great use of aria-label - i am sure this is what contributed to the 100% accessibility score